templates/index.html

```diff
@@ -1,16 +1,19 @@
 {% extends 'base.html' %}
 {% block title %}
-    Home Page
+Home Page
 {% endblock %}
 {% block content %}
-  <br/>
-  <img src="../static/images/mpc_logo.jpg" class="rounded mx-auto d-block" alt="MPC Logo">
-    <h1 class="text-center">Select Location:</h1>
-    {% for key, value in locations.items() %}
-      <div class="d-grid gap-2 col-6 mx-auto">
-        <a class="btn btn-success" href="{{ url_for('location_page', location_id=key, rack_id = '1' )}}">{{ value["name"] }}</a>
-      </div>
-      <br/>
-    {% endfor %}
-      
-{% endblock %}
\ No newline at end of file
+<br />
+<img src="../static/images/mpc_logo.jpg" class="rounded mx-auto d-block" alt="MPC Logo" />
+<h1 class="text-center">Select Location:</h1>
+{% for key, value in locations.items() %}
+<div class="d-grid gap-2 col-6 mx-auto">
+	{% if "ST" in key or "DIS" in key %}
```

Comment:
for checking if a fridge is 2d or 3d, don't reference explicit location ids, pass a variable from the view that tells the template the configuration of the fridge, what if we add new fridges in the future?. e.g. _{% if fridge == "2d" %}_